### PR TITLE
make label_image for tflite build again

### PR DESCRIPTION
**Pull request by freedomtan**
1. add namespace to label_image.h to make label_image for tflite build again
2. add --config monolithic and mention NDK settings in label_image.md
3. fix a typo in display_usage()

### CODE REVIEW

File: tensorflow/contrib/lite/examples/label_image/label_image.cc [hunk 1]
```diff
@@ -89,7 +89,7 @@ void RunInference(Settings* s) {
 
   tflite::ops::builtin::BuiltinOpResolver resolver;
 
-  tflite::InterpreterBuilder(*model, resolver)(&interpreter);
+  tflite::InterpreterBuilder (*model, resolver)(&interpreter);
```
Comment (andrehentz) [inline tensorflow/contrib/lite/examples/label_image/label_image.cc]: There should be no space here, right?

File: tensorflow/contrib/lite/examples/label_image/label_image.cc [hunk 2]
```diff
@@ -148,14 +148,22 @@ void RunInference(Settings* s) {
...
-  } else {
-    downsize<uint8_t>(interpreter->typed_tensor<uint8_t>(input), in,
-                      image_height, image_width, image_channels, wanted_height,
-                      wanted_width, wanted_channels, s);
+  switch (interpreter->tensor(input)->type) {
+    case kTfLiteFloat32:
+      s->input_floating = true;
+      downsize<float>(interpreter->typed_tensor<float>(input), in, image_height,
+                      image_width, image_channels, wanted_height, wanted_width,
+                      wanted_channels, s);
```
Comment (andrehentz) [inline tensorflow/contrib/lite/examples/label_image/label_image.cc]: please make sure the lines are less than 80-chars long and properly wrapped.

File: tensorflow/contrib/lite/examples/label_image/label_image.cc [hunk 3]
```diff
@@ -177,13 +185,21 @@ void RunInference(Settings* s) {
...
-    get_top_n<float>(interpreter->typed_output_tensor<float>(0), output_size,
-                     num_results, threshold, &top_results, s->input_floating);
-  } else {
-    get_top_n<uint8_t>(interpreter->typed_output_tensor<uint8_t>(0),
-                       output_size, num_results, threshold, &top_results,
-                       s->input_floating);
+  int output = interpreter->outputs()[0];
+  switch (interpreter->tensor(output)->type) {
+    case kTfLiteFloat32:
+      get_top_n<float>(interpreter->typed_output_tensor<float>(0), output_size,
```
Comment (andrehentz) [inline tensorflow/contrib/lite/examples/label_image/label_image.cc]: wrap line.

File: tensorflow/contrib/lite/examples/label_image/label_image.cc (hunk 1 above)
Comment (freedomtan) [reply tensorflow/contrib/lite/examples/label_image/label_image.cc]: Oops, clang-format introduced this, I guess.

File: tensorflow/contrib/lite/examples/label_image/label_image.cc (hunk 2 above)
Comment (freedomtan) [reply tensorflow/contrib/lite/examples/label_image/label_image.cc]: This is exactly 80-chars and conformed to the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html). That's why it didn't catch by clang-format.

File: tensorflow/contrib/lite/examples/label_image/label_image.cc (hunk 3 above)
Comment (freedomtan) [reply tensorflow/contrib/lite/examples/label_image/label_image.cc]: ditto

**Review by andrehentz — APPROVED** [review]

**Comment by rmlarsen** [comment]
@freedomtan Thanks!

**Pull request merged**